app/models/fs/message.rb

```diff
@@ -0,0 +1,133 @@
+class Fs::Message
```

Comment:
Implementacia je taka, ze Fs::Message neukladame do DB a nepreklapame nasledne do core domeny ako pri UPVS. Ide to rovno do core domeny a potrebne udaje navyse si ukladame v metadatach spravy.
@jsuchal je to takto vhodne alebo vidis nejaky dovod, preco by bolo k tomu lepsie pristupovat rovnako ako v pri UPVS syncu?